prov/gni/src/gnix_mr.c

```diff
@@ -42,6 +42,7 @@
 #include "gnix_mr.h"
 #include "gnix_priv.h"
 #include "common/atomics.h"
+#include "ccan/list.h"
```

Comment:
I think I'd like to continue to use the ccan lists until we get the same
functionality ported to the dlist entries. We could make this a separate
issue if you like.
-- Jim




On 6/2/15, 11:04 AM, "ztiffany" <notifications@github.com> wrote:

>In 
>prov/gni/src/gnix_mr.c
><https://github.com/ofi-cray/libfabric-cray/pull/202#discussion_r31539315>
>:
>> @@ -42,6 +42,7 @@
>>  #include "gnix_mr.h"
>>  #include "gnix_priv.h"
>>  #include "common/atomics.h"
>> +#include "ccan/list.h"
>I think we're trying to replace ccan lists with dlist_entry.
>—
>Reply to this email directly or
>view it on GitHub 
><https://github.com/ofi-cray/libfabric-cray/pull/202/files#r31539315>.
>
>